### PR TITLE
Create vikrant.json

### PR DESCRIPTION
**Pull request by 150207**
A Subdomain For Me

### CODE REVIEW

File: domains/vikrant.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "TheMalware",
```
Comment (MaskDuck) [inline domains/vikrant.json:3]: insert your gh username.

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Review by 150207 — COMMENTED** [review]
Changed gh username

File: domains/vikrant.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "shrinetojha",
+    "email": "daredevil.vikrant@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://themalware.tk"
```
Comment (MaskDuck) [inline domains/vikrant.json:7]: cname cant contain slash

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Review by 150207 — COMMENTED** [review]
Update CNAME

File: domains/vikrant.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "shrinetojha",
```
Comment (MaskDuck) [inline domains/vikrant.json:3]: this is the line for your github username.

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by 150207** [comment]
yeah ofc, i changed my gh username

**Review by MaskDuck — APPROVED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**